### PR TITLE
F/call api

### PR DESCRIPTION
**Pull request by lRebornsl**
# 🎏   In this *Milestone*  🙋‍♂️⁉:
- ✅ Use the external API Ninjas.
- ✅ Create a new component for displaying a quote.
- ✅ Fetch data inside of a new component.
- ✅ Show a loading state.
- ✅ Show an error state.
- ✅ Import the new component into your main component.
- ✅ No linters error.
- ✅ Follow the best GitHub flow.

### CODE REVIEW

File: src/components/functions/quotes.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Quote = () => {
+  const [data, setData] = useState([]);
+  const [loaded, setLoaded] = useState(false);
```
Comment (codecaiine) [inline src/components/functions/quotes.js:6]: - [ ] According to project requirements there is an error state like this `const [hasError, setHasError] = useState(false)`  Please fix in order to make your project works as expected 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @lRebornsl , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good use useEffect ✔️ 
- Ninja Quote Api is well used ✔️ 
- Quotes are well fetched ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/functions/quotes.js (hunk 1 above)
Comment (lRebornsl) [reply src/components/functions/quotes.js:6]: Solved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Anthony,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing recommended changes!
🎲 ReadMe looks great!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**